en/docs/learn/pushed-authorization-endpoint.md

```diff
@@ -0,0 +1,68 @@
+The pushed authorization request endpoint is an HTTP API at the authorization server that accepts HTTP "POST" requests with
+parameters in the HTTP request message body using the "application/x- www-form-urlencoded" format.
```

Comment:
```suggestion
parameters in the HTTP request message body using the `application/x- www-form-urlencoded` format.
```

Reply:
In doc space we do not use double quotes. Instead let's bold them (make them italics/change the formatting)